pkg/kubelet/dockertools/docker_manager.go

```diff
@@ -1130,10 +1133,12 @@ func (dm *DockerManager) checkVersionCompatibility() error {
 	return nil
 }
 
-func (dm *DockerManager) fmtDockerOpts(opts []dockerOpt) ([]string, error) {
-	version, err := dm.APIVersion()
+// getDockerOptSeparator returns the correct securityOpt separator to use for
+// the given docker version or an error.
+func (dm *DockerManager) getDockerOptSeparator() (rune, error) {
+	dockerApiVersion, err := dm.APIVersion()
```

Comment:
Hmm...why not just use `checkDockerAPIVersion()`? It uses the cached api version and also provide logic to compare versions.